src/components/formWrapper/formWrapperTabs.js

```diff
@@ -1,17 +1,18 @@
 import CrossForm from '@/components/crossForm/CrossForm'
 import NFTWrapper from '@/components/nftForm/NFTWrapper'
 import NFTWrapperCommingSoon from '@/components/nftForm/NFTWrapperCommingSoon'
+import { TOKEN_TYPE_ERC_20, TOKEN_TYPE_ERC_721 } from '@/constants/tokenType.js'
 
 const nftAvailable = process.env.VUE_APP_NFT_AVAILABLE == 'true'
 
 const formWrapperTabs = [
   {
-    id: 'erc20',
+    tokenType: TOKEN_TYPE_ERC_20,
     label: 'ERC20 Tokens',
     component: CrossForm,
   },
   {
-    id: 'erc721',
+    tokenType: TOKEN_TYPE_ERC_721,
     label: 'NFT',
```

Comment:
maybe 'ERC721 NFT' for this to be extensible in the future? (and also so users know that we only support this standard for now)

Reply:
done at https://github.com/rsksmart/tokenbridge-ui/pull/52/commits/9a00c8c01e807ba45a300cfc75b64c45df26b3f9